atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -420,6 +421,8 @@ def _get_openshift_session(self, kwargs):
 
     def get_cluster_info(self, cluster, platform):
         kwargs = deepcopy(self.config_kwargs)
+        if not self.reactor_config.is_empty():
+            kwargs['reactor_config_override'] = self.reactor_config.conf
```

Comment:
The next step here is making this backwards compatible with osbs-client versions that don't support this keyword argument. I think checking that arrangement version is `>= 6` should be sufficient.

Reply:
if osbs-client doesn't support argument it will be in kwargs and nothing will happen

Reply:
You're absolutely right! Ignore my silly request.